src/controllers/timeEntryController.js

```diff
@@ -365,10 +369,23 @@ const timeEntrycontroller = function (TimeEntry) {
     });
   };
 
+  const getCurrentDate = () => {
+    const currentDate = new Date();
+    const year = currentDate.getFullYear();
+    const month = String(currentDate.getMonth() + 1).padStart(2, "0");
+    const day = String(currentDate.getDate()).padStart(2, "0");
+    const formattedDate = `${year}-${month}-${day}`;
+
+    return formattedDate;
+  }
+
   const postTimeEntry = async function (req, res) {
+    
+    const currentDate = getCurrentDate();
```

Comment:
```suggestion
    const currentDate = moment().tz('America/Los_Angeles').format('YYYY-MM-DD');
```
Use `moment()` as used elsewhere (e.g. line 190)